### ISSUE TITLE
GBRS input file question

### ISSUE THREAD

**Issue by lawtyunc** [TheJacksonLaboratory/cs-nf-pipelines#9]
Hello all,
I hope you are doing well. I had a quick question about the GBRS pipeline. for the input file.

Input file example: 

sampleID,sex,generation,lane,fastq_1,fastq_2
Sample_42,F,42,Lane_1,/path/to/sample_42_001_R1.fastq.gz,/path/to/sample_42_001_R2.fastq.gz
Sample_42,F,42,Lane_2,/path/to/sample_42_002_R1.fastq.gz,/path/to/sample_42_002_R2.fastq.gz
Sample_101,M,10,Lane_1,/path/to/sample_101_001_R1.fastq.gz,/path/to/sample_101_001_R2.fastq.gz
Sample_10191,M,80,Lane_1,/path/to/sample_10191_001_R1.fastq.gz,/path/to/sample_10191_001_R2.fastq.gz

When you specify the generation is this just a label or does this pull from a source? If all of the mice have the same generation does it make a difference in running the pipeline?

**Comment by MikeWLloyd** [comment]
Yes, it is recommended to reanalyze the data with correct settings.

**Comment by lawtyunc** [comment]
Does the lane matter as well? If you don't have lane specific is this okay?
________________________________

**Comment by MikeWLloyd** [comment]
Lane matters if you have multiple lanes per sample. Reads are aggregated if multiple reads per unique sample name are found. The ID used in the lane field is for your own internal tracking.

**Comment by lawtyunc** [comment]
Sorry just want to clarify this to see if I need to rerun the whole GBRS pipeline from scratch?
________________________________

**Comment by MikeWLloyd** [comment]
Generation determines the genotype transition probabilities used in the GBRS model. If your DO samples are generation 51 and were run as 50, your analysis is incorrect. Outputs will differ, and your genotype determinations will be altered. 

As stated previously, the analysis should be run with correct settings.
